backend/main.py

```diff
@@ -1,14 +1,33 @@
 # Simple backend server using flask framework
 
-from flask import Flask
+from flask import Flask, jsonify, request
 from flask_cors import CORS, cross_origin
 
 app = Flask(__name__)
 CORS(app)
 
+'''
+    Get sample data from local file with a fixed output lines
```

Comment:
doc string best practice
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods